shuttle/static/assets/css/shuttle.css

```diff
@@ -1,3 +1,47 @@
+/* screen gets funky when smaller than 1247px. This changes the nav to a hamburger menu when it reaches that size. There might be a better fix */
+@media (min-width: 1247px){
+    .navbar-expand-custom {
+        flex-direction: row;
+        flex-wrap: nowrap;
+        justify-content: flex-start;
+    }
+    .navbar-expand-custom .navbar-collapse {
+        display: flex;
+    }
+    .navbar-toggler {
+        display: none;
+    }
+}
+
+@media (max-width: 1246px){
+    div.div-image {
+        margin-left: 95%;
+    }
+    .mr-auto {
+        margin-left: 5%;
+    }
+    #navbarCollapse a{
+        display: block;!important;
+        border-bottom: 1px solid #EAEAEB;
+    }
+    .navbar-toggler {
+        position: absolute;
+        top: 10px;
+    }
+    .row {
+        display: block;
+    }
+}
```

Comment:
It would be worth it to either change the pixel (px) amounts to em or % or just use the built in bootstrap classes to shrink to the hamburger menu at specific times. Pixel is a static number where em/%/bootstrap are dynamic so they can change based on the size of the window versus how many pixels the screen has.